releasenotes/notes/celery-redbeat-a3b6b7a6131c33d3.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    tracing: This introduces automatic instrumentation for the celery-redbeat celery task scheduler.
```

Comment:
note: `redbeat` needs to be added to the `spelling_wordlist.txt` to pass the build_docs job.